base/compiler/compiler.jl

```diff
@@ -14,6 +14,7 @@ const setproperty! = Core.setfield!
 const swapproperty! = Core.swapfield!
 const modifyproperty! = Core.modifyfield!
 const replaceproperty! = Core.replacefield!
+const _DOCS_ALIASING_WARNING = ""
```

Comment:
```suggestion
```
Looks like an unrelated change?

Reply:
True. We should land it eventually, as it is required for loading a testing version of inference now, but it isn't connected to any particular PR.